stream/basic_rtmp_videostream.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type BasicRTMPVideoStream struct {
-	streamID     string
+	opaque       AppData // opaque app-supplied data
```

Comment:
Consider naming this `appData`? IMO it isn't that clear what `opaque` is when reading the rest of the code

Reply:
Fixed